app/tests/Feature/Http/VotingLists/JsonTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+use App\Enums\CountryEnum;
+use App\Enums\VoteResultEnum;
+use App\Group;
+use App\Vote;
+use App\Member;
+use App\VotingList;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Carbon;
+
+uses(Tests\TestCase::class, RefreshDatabase::class);
+
+beforeEach(function () {
+    $date = new Carbon('2021-05-23');
+    $display_title = 'Budged 2020';
+
+    $greens = Group::factory([
+        'code' => 'GREENS',
+        'name' => 'Greens/European Free Alliance',
+        'abbreviation' => 'Greens/EFA',
+    ])->create();
+
+    $this->greensFor = Member::factory([
+        'first_name' => 'Jane',
+        'last_name' => 'DOE',
+        'country' => CountryEnum::NL(),
+    ])->activeAt($date, $greens)->count(1);
+
+    $this->votingList = VotingList::factory([
+        'date' => $date,
+        'vote_id' => Vote::factory([
+            'result' => VoteResultEnum::REJECTED(),
+        ]),
+        'description' => 'Budged 2020',
+        ])
+        ->withMembers('FOR', $this->greensFor)
+        ->create();
+});
+
+it('creates a JSON with one row per member', function () {
+    $response = $this->get("/votes/{$this->votingList->id}.json");
+
+    $decoded = json_decode($response->content(), true);
+
+
+    $expected = [
+        'title' => "Children’s rights",
```

Comment:
@tillprochaska we are a little bit confused here.  Where is the title `Children’s rights` coming from ? We acutely expected `Budged 2020` as a title 🙃 .

Reply:
You’re creating a record using a `VoteFactory`. A `VoteFactory` will by default create an associated `VoteCollection` record using the `VoteCollectionFactory` (see the [factory definition](https://github.com/HowTheyVote/epvotes/blob/main/app/database/factories/VoteFactory.php#L31)). And the `VoteCollectionFactory` in turn has a default title (in this case "Children’s rights". The whole factory setup is a little confusing, I’ll have to admit that!

So if you want to control the display title, you can configure the `VoteCollectionFactory` yourself like this:

```php
$this->votingList = VotingList::factory([
  'date' => $date,
  'vote_id' => Vote::factory([
      'result' => VoteResultEnum::REJECTED(),
      'vote_collection_id' => VoteCollection::factory([
        'title' => 'Budget 2020',
      ]),
  ]),
  'description' => 'Budged 2020',
])->withMembers('FOR', $this->greensFor)->create();
```